recipe/meta.yaml

```diff
@@ -21,7 +21,7 @@ requirements:
   run:
     - python >=3.9
     - numpy
-    - pint =0.21
+    - pint >=0.21,<0.23
```

Comment:
pint 0.22 is explicitly tested, and openff-units 0.2.1 release notes say that pint 0.22 is supported.

Reply:
Hmm, notably one of them has 3.8 in the traceback. I need to add tests (in the repo or here) to see if the issue is also in 3.9 and 3.10. Notably I'm not permitted to drop 3.9 until next month

Reply:
A note for myself: I think with this change (long ago) in, we should be able to loosen this constrain throughout our infrastructure. I'll need to create several test environments to make sure this doesn't cause issues, since this is a fulcrum that can crumble our entire infrastructure and it's tricky to test against.

Reply:
https://github.com/openforcefield/openff-toolkit/pull/1640/files#diff-427be19ea8f280b778bf2a9be82bee13a62d544860217c096e8ede75c3351d22L910